Sources/ZippyJSON/ZippyJSONDecoder.swift

```diff
@@ -167,42 +164,66 @@ public final class ZippyJSONDecoder {
         }
     }
 
+    @Synchronized
+    private var _userInfo: [CodingUserInfoKey : Any] = [:]
+    public var userInfo: [CodingUserInfoKey : Any] {
+        get { _userInfo }
+        set { _userInfo = newValue }
+    }
 
-    public var userInfo: [CodingUserInfoKey : Any] = [:]
-
-    public var nonConformingFloatDecodingStrategy: NonConformingFloatDecodingStrategy
+    @Synchronized
+    private var _nonConformingFloatDecodingStrategy: NonConformingFloatDecodingStrategy
+    public var nonConformingFloatDecodingStrategy: NonConformingFloatDecodingStrategy {
```

Comment:
why can't public var have `@Synchronized` on it and we don't need to wrap? is it to avoid exposing `@Synchronized`?